dspace/config/dspace.cfg

```diff
@@ -1053,6 +1053,10 @@ webui.preview.brand.fontpoint = 12
 
 ##### Settings for item count (strength) information ####
 
+# Whether to display collection and community strengths (i.e. item counts)
+#The default behaviour is to show item counts.
+#webui.strengths.show = true
```

Comment:
Again, I'd recommend this be false by default.  Same as in 6.x.  Once we are able to better test performance we can turn them on by default at a later date.

Reply:
Done.